### PR TITLE
Update README.md. Add your own @s to CREDITS.md

### PR DESCRIPTION
**Pull request by ethanonweller**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,15 @@
-# collaborative-pixel-art
-A terrible idea that was shamelessly stolen from Reddit
+# Collaborative Pixel Art
+### A terrible idea that was shamelessly stolen from Reddit
+
+We thought it was unfair that [r/place](https://www.reddit.com/r/place/) is only up for a few days at a time, so we are in the process of creating our own. 
+
+## Contributors
+[Ethan Onweller](https://github.com/ethanonweller)
```
Comment (cbpudding) [inline README.md]: Would prefer a CREDITS file for this. No need to put this in the README.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,15 @@
...
+
+## TODO
+- [ ] User authentication
+- [ ] Incorporate DB and ORM
+- [ ] Frontend design
+- [ ] Zoom/Scaling
+
+...etc
```
Comment (cbpudding) [inline README.md]: This part can probably be replaced by GitHub projects. Let me know in the group chat if you want to start using that.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,15 @@
-# collaborative-pixel-art
-A terrible idea that was shamelessly stolen from Reddit
+# Collaborative Pixel Art
+### A terrible idea that was shamelessly stolen from Reddit
```
Comment (cbpudding) [inline README.md]: Probably doesn't need to be a header. It was fine as-is.

**Review by cbpudding — CHANGES_REQUESTED** [review]

**Review by LieutenantLark — DISMISSED** [review]
yes

File: README.md (hunk 3 above)
Comment (ethanonweller) [reply README.md:2]: It just looks nicer as a header imo

File: README.md (hunk 1 above)
Comment (ethanonweller) [reply README.md]: 👍

**Review by cbpudding — APPROVED** [review]
Looks good! Clear to merge!

**Review by LieutenantLark — APPROVED** [review]

**Pull request merged**